### PR TITLE
added content

### PR DESCRIPTION
**Pull request by JL347**

### CODE REVIEW

File: js-intro.js [hunk 1]
```diff
@@ -6,17 +6,41 @@ var mantra = "Be the dev";
...
+
 // Stretch: Write the code that returns the position of 'v' in mantra.
 
+console.log(mantra.indexOf("v"))
 
 // Consider the following statement:
 
-var codingIsEasy
-var imBored
+var codingIsEasy = "Give me a challenge!"
```
Comment (JzGo) [inline js-intro.js:42]: Try giving these variables boolean values

File: js-intro.js [hunk 2]
```diff
@@ -26,10 +50,12 @@ if (codingIsEasy && imBored) {
 
 // 4. Update the codingIsEasy and imBored variables above the statement to the values required to log "I need more!"
 
+
 // Consider the following statement:
 
 var imLost
```
Comment (JzGo) [inline js-intro.js:56]: Again, consider giving these variables boolean values

File: js-intro.md [hunk 3]
```diff
@@ -3,13 +3,26 @@
...
+    .toUpperCase
+    .toLowerCase
+    .charAt()
+    .concat()
 
 #### 4. Explain the difference between accessor and mutator methods. Why would you use one over the other?
+    an accessor is a checker and a mutator is a changer. you use accessor's to access information in a function and you use mutators to change said information.
 
 #### 5. Explain the differences between console.log() and return. Why would you use one over the other?
+    console.log() is the method that is used to log the entire written code into terminal
```
Comment (JzGo) [inline js-intro.md:23]: This is a good start! Consider this, if you wanted to use the result of one function in another part of your program or in another function, which would you use?

File: js-intro.js [hunk 4]
```diff
@@ -6,17 +6,41 @@ var mantra = "Be the dev";
 
 // 1. Write the code that returns 'B' from mantra.
 
+mantra = (str) => {
```
Comment (JzGo) [inline js-intro.js:9]: I like where you were going with this! How would you tell the function to search for the string that you passed as an argument to your mantra function?

File: js-intro.md [hunk 5]
```diff
@@ -3,13 +3,26 @@
...
+    return is used to return a value in a specific piece of code
+    you would use console.log() to see if your code works and return would be used to see a specific value
 
 #### 6. What is one thing we did in class this week that helped you learn the most?  
+    some of the more extensive examples were helpful for me. i'm a visual learner so seeing someone write out a bunch of examples help me understand what i'm learning.
```
Comment (JzGo) [inline js-intro.md:28]: Thanks for this! We'll keep this in mind!

**Review by JzGo — COMMENTED** [review]
Solid work so far! I'd like you to take a stab at js question number 7 and implement what I suggest in my comments. Once you're done, let me know on slack and I will take another look.

File: js-intro.js (hunk 1 above)
Comment (JL347) [reply js-intro.js:42]: var codingIsEasy = true
var imBored = true

File: js-intro.js (hunk 2 above)
Comment (JL347) [reply js-intro.js:56]: var imLost = false
var imFrustrated = false

File: js-intro.js [hunk 6]
```diff
@@ -51,12 +83,14 @@ var message = "thisisateststring"
 
 // 7. Write a function that takes a string like message as an argument and returns the string without vowels.
 
+
```
Comment (JL347) [inline js-intro.js:86]: function noVowels(letters){
  var vowels = ["a", "e", "i", "o", "u"]
  let newString = []

  for(let i=0; i<letters.length; i++){
    if(vowels.indexOf(letters[i]) === -1){
      newString.push(letters[i])
    }
  }
  return newString
}

console.log(noVowels(message))

Comment (JL347) [inline js-intro.js:86]: function noVowels(letters){
  var vowels = ["a", "e", "i", "o", "u"]
  let newString = []

  for(let i=0; i<letters.length; i++){
    if(vowels.indexOf(letters[i]) === -1){
      newString.push(letters[i])
    }
  }
  return newString
}

console.log(noVowels(message))

File: js-intro.js [hunk 7]
```diff
@@ -51,12 +83,14 @@ var message = "thisisateststring"
...
 // Stretch: Update your function to throw an exception if the argument is not a string.
 
 // Super Duper Stretch: Update your function to throw a TypeError if the argument is not a string.
 
 // Consider the following variable:
 
-var toonimals = [ {name: 'Itchy', animal: 'mouse'}, {name: 'Stimpy', animal: 'cat'}, {name: 'Daffy', animal: 'duck'}, {name: 'Scratchy', animal: 'cat'}, {name: 'Ren', animal: 'dog'}, {name: 'Felix', animal: 'cat'}]
+// var toonimals = [ {name: 'Itchy', animal: 'mouse'}, {name: 'Stimpy', animal: 'cat'}, {name: 'Daffy', animal: 'duck'}, {name: 'Scratchy', animal: 'cat'}, {name: 'Ren', animal: 'dog'}, {name: 'Felix', animal: 'cat'}]
 
-// 8. Write a function that takes an array like toonimals and returns an array with only the toons that are cats.
```
Comment (JL347) [inline js-intro.js:62]: onlyCats = (str) => {
  var newCats = []
  for(let i = 0; i < str.length; i++){
    if(str[i].animal === 'cat'){
      newCats.push(str[i])
    }
  }
  return newCats
}

console.log(onlyCats(toonimals))

File: js-intro.md (hunk 3 above)
Comment (JL347) [reply js-intro.md:23]: you would use return. Ultimately, if you want the same result from one function to happen in a different one than returning that same value will solve it.

File: js-intro.js (hunk 4 above)
Comment (JL347) [reply js-intro.js:9]: you could use a for loop in this situation.

for (let i = 0; i < str.length; i++)

I hope that answers the question! Let me know if there's a better way to do so.

File: js-intro.js (hunk 1 above)
Comment (JzGo) [reply js-intro.js]: Great!

**Comment by JzGo** [comment]
This is great Jared! Go ahead and merge.

It might be easier to do assessments through AWS if your Windows environment is giving you trouble. The process is about the same as it is in the classroom. So for next assessment, try firing it up on AWS. If it's still a problem, reach out.

**Pull request merged**